### DISCUSSION TITLE
Wallets Support

### DISCUSSION THREAD

**Discussion by LilFatFrank** [family/connectkit#180] [General]
Is there a way for me to integrate multiple wallets
Something like frontier wallet extension?

**Comment by lochie** [comment]
We're [working on supporting adding custom wallets and extensions](https://github.com/family/connectkit/pull/184), but if you want to have a wallet or extension shipped with ConnectKit you are welcome to make a PR with additional wallet configurations (see [frontier](https://github.com/family/connectkit/blob/main/packages/connectkit/src/wallets/connectors/frontier.tsx) for an example).

If you're looking to bring in the frontier extension, you can add an `installed` prop to this config with the correct check to see if it's installed (see [Brave](https://github.com/family/connectkit/blob/90f73491c980fbfdc2be4e9cea5cf84297c6599a/packages/connectkit/src/wallets/connectors/brave.tsx#L7) for an example).
